includes/documents/class-wcpdf-invoice.php

```diff
@@ -456,6 +459,48 @@ public function get_date_title() {
 		return apply_filters( "wpo_wcpdf_{$this->slug}_date_title", $date_title, $this );
 	}
 
+	/**
+	 * Schedules the invoice yearly reset number with AS
+	 */
+	public function schedule_yearly_reset_number() {
+		// checks AS functions existence
+		if( ! function_exists( 'as_schedule_single_action' ) || ! function_exists( 'as_get_scheduled_actions' ) ) {
```

Comment:
is there a reason you're using AS instead of WP_Cron? This is not a volumous task.

Reply:
Because WC has it and our plugin depends on WC? Also is more easy to check status, history, etc, without installing a new plugin.

Reply:
ok, fair enough, the logging can indeed be helpful